bin/control.sh

```diff
@@ -105,60 +125,25 @@ fi
 
 #
 # Uncomment to set preference for IPv4 stack.
-#
-# CONTROL_JVM_OPTS="${CONTROL_JVM_OPTS} -Djava.net.preferIPv4Stack=true"
-
-#
-# Assertions are disabled by default since version 3.5.
-# If you want to enable them - set 'ENABLE_ASSERTIONS' flag to '1'.
-#
-ENABLE_ASSERTIONS="1"
-
-#
-# Set '-ea' options if assertions are enabled.
-#
-if [ "${ENABLE_ASSERTIONS:-}" = "1" ]; then
-    CONTROL_JVM_OPTS="${CONTROL_JVM_OPTS} -ea"
-fi
 
```

Comment:
Let's stick with the same style, i.e.

#
# BlaBlaBla
#
# CONTROL_JVM_OPTS="${CONTROL_JVM_OPTS} -Djava.net.preferIPv4Stack=true"

Reply:
I've added a `# `at the start of line 129 (formerly line 128), which was unintentionally missing. Thanks for pointing that out. Could you please confirm if this was the intended correction?